### PR TITLE
fixes #6881 -  check if in rake when validating template changes

### PR DESCRIPTION
**Pull request by stbenjam**

### CODE REVIEW

File: test/unit/config_template_test.rb [hunk 1]
```diff
@@ -112,6 +112,16 @@ class ConfigTemplateTest < ActiveSupport::TestCase
     refute_valid tmplt, :base, /Katello/
   end
 
+  test "should change a locked template while in rake" do
+    Foreman.stubs(:in_rake?).returns(true)
+    tmplt = config_templates(:locked)
+    tmplt.template = "changing the template content"
+    tmplt.name = "giving it a new name too"
+    assert_equal tmplt.locked, true
```
Comment (domcleal) [inline test/unit/config_template_test.rb]: `assert tmplt.locked`

File: test/unit/config_template_test.rb [hunk 2]
```diff
@@ -112,6 +112,16 @@ class ConfigTemplateTest < ActiveSupport::TestCase
...
+    assert tmplt.valid?
```
Comment (domcleal) [inline test/unit/config_template_test.rb]: assert_valid tmplt

File: test/unit/config_template_test.rb [hunk 3]
```diff
@@ -112,6 +112,16 @@ class ConfigTemplateTest < ActiveSupport::TestCase
...
+    assert tmplt.save!
```
Comment (domcleal) [inline test/unit/config_template_test.rb]: unnecessary if you're only testing validation, removing it will speed up tests

**Comment by stbenjam** [comment]
@domcleal Fixed

**Comment by domcleal** [comment]
Merged as ad2c9aa, thanks @stbenjam!

**Pull request closed without merging**